great_expectations/data_context/data_context.py

```diff
@@ -406,7 +406,8 @@ def get_docs_sites_urls(
         for _site_name, site_config in sites.items():
             site_builder = self._load_site_builder_from_site_config(site_config)
             url = site_builder.get_resource_url(resource_identifier=resource_identifier)
-            site_urls.append({"site_name": _site_name, "site_url": url})
+            if url is not None:
+                site_urls.append({"site_name": _site_name, "site_url": url})
```

Comment:
I believe this should still return with None as the url so that a consuming system can be aware of a configured site.

Reply:
None would result in localsite=None as an example

Reply:
But the None case empty string url added.